### ISSUE TITLE
`nix develop` fails on example `python-local-development` (dependency does not exist)

### ISSUE THREAD

**Issue by akaihola** [nix-community/dream2nix#1007]
I get the following error when running `nix develop` on current `main` in the `examples/packages/languages/python-local-development/` directory:
```shell
[~/dream2nix/examples/packages/languages/python-local-development]$ nix develop
Error: The python dependency charset-normalizer of mytool is configured to be installed in editable mode,
but the provided source location
/home/akaihola/dream2nix/examples/packages/languages/python-local-development/.editables/charset_normalizer
does not exist.
Please provide a path to a local copy of the source code of charset-normalizer.
```

I can't find documentation on how I'm supposed to manage editable dependencies. Do I need to download the source distribution for each of the editable dependencies (just `charset-normalizer` in this case) and extract them into `./.editables/`? Or is there some tooling to manage this? It would be helpful if this was pointed out in the documentation and in the error message.

**Comment by purepani** [comment]
> Do I need to download the source distribution for each of the editable dependencies (just charset-normalizer in this case) and extract them into ./.editables/?
Yeah this is how it works atm. The editables module is pretty new so there's probably still some work/tooling that can make it a bit smoother.
That said, i'm not quite sure why we don't have the charset-normalizer package in the example repo itself so that it works out of the box. That is something we can add.
